cluster-autoscaler/cloudprovider/clusterapi/deployment.yaml

```diff
@@ -0,0 +1,158 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cluster-autoscaler
+  namespace: kube-system
+  labels:
+    app: cluster-autoscaler
+spec:
+  selector:
+    matchLabels:
+      app: cluster-autoscaler
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: cluster-autoscaler
+    spec:
+      containers:
+      - image: us.gcr.io/k8s-artifacts-prod/autoscaling/cluster-autoscaler:v1.18.1
```

Comment:
I'm not quite sure how we want to handle it yet, but we probably want to consider taking into account for overriding either the image repository/name and the version tag (or both), since users may want to override the registry/image name for testing non-released versions and the version tag will need to correspond to the same minor kubernetes release that is running on the managed workload cluster.

Reply:
this makes way too much sense ;)